### PR TITLE
zetcd: fix stat data for root directory

### PR DESCRIPTION
**Pull request by heyitsanthony**
Consolidates NoNode checking, adds special root Cversion handling.

### CODE REVIEW

File: stat.go [hunk 1]
```diff
@@ -64,5 +64,21 @@ func statTxn(txnresp *etcd.TxnResponse) (s Stat) {
...
+		return s, nil
+	}
+
+	// fix ups for special root node
+
+	// lie about having the quota dir so stat on "/" xchks OK
+	s.NumChildren++
+	// Cversion initialized to -1
+	if len(cver.Kvs) == 0 {
+		s.Cversion--
```
Comment (xiang90) [inline stat.go]: why not just set to -1?

File: zketcd.go [hunk 2]
```diff
@@ -740,3 +737,5 @@ func mkErrTxnOp(err error) opBundle {
 		},
 	}
 }
+
+func isNoNode(path string, s Stat) bool { return path != "/" && s.Ctime == 0 }
```
Comment (xiang90) [inline zketcd.go]: where do we use this func?

File: stat.go (hunk 1 above)
Comment (heyitsanthony) [inline stat.go]: yes that works better; len(cver.Kvs) == 0 => cversion == 0

File: zketcd.go (hunk 2 above)
Comment (heyitsanthony) [inline zketcd.go]: removed; opened issue about unused linting

**Comment by xiang90** [comment]
lgtm

**Pull request merged**